.rufo

```diff
@@ -0,0 +1 @@
+quote_style :single
```

Comment:
Yeah, I see the point.

I could add this easily to my ignores, but then other people that have Rufo installed will have the same problem. Since the code style is not the default style of Rufo potential contributors will have to fight with Rubocop all the time leading to frustration and to less contributions.

Since this file is not harming us in any means (and the "out-of-sync" issue is neglectable), but quite contrary will help contributions I would still say let's keep it.

My 2 cents, but if you want, I will remove this.